src/frontend/components/render-state/render-state.html

```diff
@@ -4,19 +4,37 @@
   <span *ngIf="none">
     No state to show
   </span>
-  <span
+  <span class="property-container"
      (click)="expandTree(k, $event)"
      *ngIf="nest(k)">
-    <span class="info-key">
+    <span class="info-key" [ngClass]="{output: classification(k) === StateClassification.Output}">
       <div [ngClass]="{
         expander: true,
         rotate90: !expanded(k),
-        transparent: keys(state[k]).length == 0
+        transparent: keys(state[k]).length === 0,
+        'display-none': hideExpander(k)
       }"></div>
-      <span *ngIf="level === 0 && inputs[k]" class="primary-color">@Input(<span class="info-value" [hidden]="!inputs[k].alias">'{{ inputs[k].alias }}'</span>)&nbsp;</span>{{k}}:
+      <span [ngSwitch]="classification(k)" class="primary-color">
```

Comment:
You should be very careful with whitespace here, especially if there might be a mismatch to the equivalent whitespace in the StateValues template.